.travis.yml

```diff
@@ -4,7 +4,6 @@ scala:
   - 2.11.2
 cache:
   directories:
-  - idea
```

Comment:
I am not so happy to disable that. However I think it will bite us again and again. Would it be possible to cache the zip files but ignore the extracted folders ?